GeosCore/drydep_mod.F90

```diff
@@ -1225,6 +1225,10 @@ SUBROUTINE DEPVEL( Input_Opt, State_Chm, State_Diag, State_Grid, &
     ! Small number
     REAL(fp), PARAMETER :: SMALL = 1.0e-10_f8
 
+    ! VTS
```

Comment:
Add description of variable here.  Also the difference between `VTSoutput` and `VTSouptut_` isn't clear to me (other than one is a scalar and another is an array).

Reply:
Your understanding is correct! VTS stands for gravitational settling velocity which is a component of aerosol dry deposition velocity. The variable VTSoutput is used to store calculated settling velocity by function AERO_SFCRSII, ADUST_SFCRSII and DUST_SFCRSII for a specific aerosol species and a specific land type. And VTSoutput_ is used to collect settling velocities for all deposited aerosol species and all land types.